README.md

```diff
@@ -866,6 +866,10 @@ This library supports the standard [Atlassian supported browsers](https://conflu
 
 Currently mobile is not supported. However, there are plans to add touch support in the future
 
+## Translation
+This library is also available in other languages:
+- ![kr](https://raw.githubusercontent.com/gosquared/flags/master/flags/flags/shiny/24/South-Korea.png) **Korean**: [leehyunggeun/react-beautiful-dnd](https://github.com/LeeHyungGeun/react-beautiful-dnd-kr)
```

Comment:
Could the fork of the repo just include the `README.md` rather than the source also? What do you think?